content/en/news/release-notes.md

```diff
@@ -6,6 +6,30 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+## 1.43.0
+Sprint Release: November 12nd, 2021
+
+Features:
+
+* [Allow Kiali Graphs to show EgressGateway traffic to ServiceEntry](https://github.com/kiali/kiali/issues/3765)
+* [(Feature Request) Support mounting existing secret into Kiali Pod](https://github.com/kiali/kiali/issues/4468)
+* [Calculate graph importance score](https://github.com/kiali/kiali/issues/2888)
+* [Validations - Ensure ServiceEntry has WorkloadEntry addresses](https://github.com/kiali/kiali/issues/4339)
+* [Support getting the root namespace from Istio configuration](https://github.com/kiali/kiali/issues/4448)
+* [ingress created by Kiali CR does not include ingress class - need new deployment.ingress setting](https://github.com/kiali/kiali/issues/4342)
```

Comment:
I think the following should appear after this line 19:

===

Please note this introduces a backward-incompatible change. It requires a minor change to an existing Kiali CR. The previous ingress settings were:

```yaml
deployment:
  ingress_enabled: <true|false>
  override_ingress_yaml:
    ...the override yaml here...
```

This has been changed to the following:

```yaml
deployment:
  ingress:
    enabled: <true|false>
    override_yaml:
      ...the override yaml here...
```